src/frontend/apps/impress/src/layouts/MainLayout.tsx

```diff
@@ -49,11 +49,13 @@ export function MainLayout({
               ? colorsTokens['greyscale-000']
               : colorsTokens['greyscale-050']
           }
+          $position="relative"
           $css={css`
             overflow-y: auto;
             overflow-x: clip;
           `}
         >
+          <DocCreationOverlay />
```

Comment:
Same I think it could be nice to have something more generic, that can be used for other things that `DocCreation`. 
The problem that I find as well, is I get a bit the flickering effect, if we could implement a light transition before unmounting the skeleton would be great (fadeOut / fadeIn ?)